### DISCUSSION TITLE
How to add description or About to the bot?

### DISCUSSION THREAD

**Discussion by lifeofsoumya** [eritislami/evobot#1031] [Q&A]
If I want to add the invite link or any other text apart from 'Listening to play and help", is it possible?

**Comment by Shaxpartan** [comment]
Yes,  go index.js then at line 27 you will see  client.user.setActivity(`${PREFIX}help and ${PREFIX}play`, { type: "LISTENING" });  You can change the activity example client.user.setActivity(`You Status Here`, { type: "LISTENING" });  also if you want you can change type to LISTENING to PLAYING..
